packages/compas-open-plugin/src/compas-open-plugin.ts

```diff
@@ -40,22 +40,29 @@ import {
   export default class CompasOpenMenuPlugin extends LitElement {
     @query("mwc-dialog#compas-open-dlg")
     dialog!: Dialog;
+    /** if true, the user can select a local file to open */
```

Comment:
Above the class I see that its annotated with param. This should actually be prop